manager/burrow-mint/state/execution.go

```diff
@@ -430,8 +430,10 @@ func ExecTx(blockCache *BlockCache, tx txs.Tx, runCall bool, evc events.Fireable
 				return txs.ErrTxInvalidAddress
 			}
 			// check if its a native contract
-			if vm.RegisteredNativeContract(LeftPadWord256(tx.Address)) {
-				return fmt.Errorf("NativeContracts can not be called using CallTx. Use a contract or the appropriate tx type (eg. PermissionsTx, NameTx)")
+			if vm.RegisteredNativeContractAddress(tx.Address) {
```

Comment:
please revert `LeftPadWord256`; such conversion from []byte to Address should be handled by a type `Address`, not by a function signature.  Until we introduce such stronger typing the paradigm is to use these conversion functions.